docs/UserGuide.md

```diff
@@ -610,12 +610,32 @@ Back to [Table of Contents](#table-of-contents).
 
 ### Adding an appointment : `add_a`
 
-Allows you to add an appointment into the system.
+Allows you to add an appointment to the system.
 
 **Format**: `add_a pic\PATIENT_NRIC dic\DOCTOR_NRIC from\START_TIME to\END_TIME`
 
 **Examples**:
-* `add_a pic\S9912343G dic\T0212385J from\2023-09-11 07:30 to\2023-09-11 08:00 `
+* `add_a pic\S9912343G dic\T0212385J from\2023-09-11 07:30 to\2023-09-11 08:00`
+* `add_a from\2023-09-12 07:30 to\2023-09-12 08:00 dic\T0212385J pic\S9912343G`
+
+**Sample Usage**:
+1. Assuming you want to add an appointment between a patient "John Doe" with NRIC "T0212385J" and doctor "Betsy Crowe" with NRIC "S9912343G" starting from "2023-09-13 12:00" ending at "2023-09-13 12:30".
+
+2. Enter the following command:
+```
+add_d pic\T0212385J dic\S9912343G from\2023-09-13 12:00 to\2023-09-13 12:30
+```
+
+3. The result box will display the following message:
+```
+New appointment added: Patient NRIC: T0212385J; Doctor NRIC: S9912343G; From: 2023-09-13 12:00; to: 2023-09-13 12:30
+```
+
+4. You have successfully added the appointment into the database.
+
+<div markdown="span" class="alert alert-warning">:exclamation: <b>Caution:</b>
+Do ensure that there exists a patient and doctor with the NRIC that you input into this command!
```

Comment:
Alternative phrasing `Do ensure that the patient and doctor you are entering exist in the database!`